package.json

```diff
@@ -153,12 +153,14 @@
 		"@testing-library/react": "11.2.6",
 		"@testing-library/react-hooks": "3.7.0",
 		"@testing-library/user-event": "12.8.3",
+		"@types/classnames": "^2.2.11",
```

Comment:
FYI @samueljseay this package doesn't contain any types, and the latest ~2.3.1 version of classnames comes with types: https://github.com/JedWatson/classnames/blob/master/HISTORY.md (just realized this myself).

Reply:
oh awesome. ok, thanks I'll fix this.

Reply:
Fixed in https://github.com/woocommerce/woocommerce-admin/pull/6787/commits/f755dd4696491bf008d20f3958fe432e4b553d49